src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,44 @@
+.stars--4 {
+  display: flex;
+  padding-top: 16px;
+
+  & .stars__star:nth-child(1) {
+    display: block;
+    background-image: url(../images/star-active.svg);
+    width: 13px;
+    height: 12.2px;
+    margin-left: 17px;
```

Comment:
The `margin-left` for the first star is 17px, which is not a multiple of 4. This does not follow the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). You should consider using a value that is a multiple of 4.